libminifi/src/core/FlowConfiguration.cpp

```diff
@@ -174,4 +181,22 @@ std::shared_ptr<core::controller::ControllerServiceNode> FlowConfiguration::crea
   return controllerServicesNode;
 }
 
+std::string FlowConfiguration::decryptProperty(const std::string& encrypted_value) const {
+  static const utils::Regex is_encrypted{R"(enc\{.*\})"};
+  if (!utils::regexMatch(encrypted_value, is_encrypted)) {
```

Comment:
It may be premature optimization, but I'd avoid regex here, and use `starts_with("enc{")` and `ends_with('}')` instead.

Reply:
good idea, fixed in e3c2ca4db93eed4e6ea18b49662bc16be1fb2477